source/system/system.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"github.com/kubernetes-incubator/node-feature-discovery/source"
+)
+
+// Source implements FeatureSource.
+type Source struct{}
+
+// Name returns an identifier string for this feature source.
+func (s Source) Name() string { return "system" }
+
+// Discover returns feature names for system configuration: irqbalance, swap, ksm, memory compaction, iommu...
+func (s Source) Discover() (source.Features, error) {
+	features := source.Features{}
+	active := SystemctlStatusIsActive("irqbalance")
```

Comment:
SystemctlStatusIsActive() should return error if it is not able to detect the status of the service. In case of error, no label should be set  as we don't really know the status, if it's *false* or *true*. This might happen e.g. if the mounts are missing from the container spec.

Reply:
Done.